### PR TITLE
[@types/ua-parser-js] issue #25202 fix

### PR DESCRIPTION
**Pull request by KseniaKapnik**
updating export declaration to be the standard ES6/TypeScript
@MeLlamoPablo

### CODE REVIEW

**Review by MeLlamoPablo — APPROVED** [review]

File: types/ua-parser-js/index.d.ts [hunk 1]
```diff
@@ -139,7 +139,7 @@ declare namespace IUAParser {
 }
 
 declare module "ua-parser-js" {
-    class UAParser {
+    export class UAParser {
```
Comment (uniqueiniquity) [inline types/ua-parser-js/index.d.ts]: This doesn't seem right to me.
In particular, the module _is_ the class (i.e. it's a constructable module) as shown in [the source](https://unpkg.com/ua-parser-js@0.7.19/src/ua-parser.js).
However, this change makes the type say that the module is an object with a property `UAParser`, which will fail at runtime.

If you want to use ES6 imports (i.e. `import UAParser from 'ua-parser-js';`), you can use the `--esModuleInterop` compiler setting. This doesn't seem right to me.
In particular, the module _is_ the class (i.e. it's a constructable module).
However, this change makes the type say that the module is an object with a property `UAParser`, which will fail at runtime.

If you want to use ES6 imports (i.e. `import UAParser from 'ua-parser-js';`), you can use the `--esModuleInterop` compiler setting.

**Review by uniqueiniquity — CHANGES_REQUESTED** [review]

File: types/ua-parser-js/index.d.ts (hunk 1 above)
Comment (MeLlamoPablo) [reply types/ua-parser-js/index.d.ts]: It doesn't fail at runtime though. There is this line in the source: `exports.UAParser = UAParser;`, so the module is the class, but also contains the class, which makes this syntax valid.

If you want, you can take a look at #25202 for more context on this PR.

Comment (uniqueiniquity) [reply types/ua-parser-js/index.d.ts]: Oops, my mistake! I missed that piece.

**Review by uniqueiniquity — APPROVED** [review]

**Comment by uniqueiniquity** [comment]
Congratulations on your first DefinitelyTyped contribution!
Thank you for being a part of the community!

**Pull request merged**